src/main.rs

```diff
@@ -1,40 +1,134 @@
 extern crate clap;
 extern crate ignore;
+extern crate regex;
+extern crate difference;
+extern crate term;
 
 use clap::{App, Arg};
 use ignore::Walk;
+use std::path::Path;
+use std::fs::File;
+use std::io::prelude::*;
+use regex::Regex;
+use difference::{Changeset, Difference};
+
+#[allow(unused_must_use)]
```

Comment:
maybe add a comment explaining why you need this ?

Reply:
I .. dont know. It's copy paste.

it's not production ready code, I believe that we have to write our own replacer module.

Reply:
is it fine with it?

The code has to be reworked anyway. (it has been moved to a replacer submodule)

Reply:
fixed